crates/sui-core/src/consensus_adapter.rs

```diff
@@ -402,6 +396,35 @@ impl ConsensusAdapter {
         )
     }
 
+    fn await_submit_delay_user_transaction(
+        &self,
+        committee: &Committee,
+        tx_digest: &TransactionDigest,
+    ) -> (Duration, usize, bool) {
+        let (mut position, mapped_to_low_scoring) = self.submission_position(committee, tx_digest);
+
+        const MAX_LATENCY: Duration = Duration::from_secs(5 * 60);
+        const DEFAULT_LATENCY: Duration = Duration::from_secs(5);
```

Comment:
Let's update DEFAULT_LATENCY to 2s here or in another PR.

Reply:
I would prefer to address this latency change separately as is more cross-network concern